### DISCUSSION TITLE
Use percent as y axis?

### DISCUSSION THREAD

**Discussion by roblyle** [highfestiva/finplot#355] [Ideas]
Hi,

Firstly thanks for all your work on this fantastic library!

I was wondering whether (and how) it is possible to express the y-axis in percent rather than the absolute stock price. Binance offer this option on their candlestick trading view and it is useful to judge the shift in prices.

Thanks for your help.

Robin

**Comment by highfestiva** [comment]
Hi, that feature is for manual judgement of how things are moving. finplot is primarily for backtesting, but I added a function, `_create_axis()`, that you can override and implement your own y-axis: a96718e. It's not hard at all, you only need to override the YAxisItem class to set your own tick strings. GL!

**Comment by roblyle** [comment]
Hi and thanks for this input. I will have an experiment with this and see if I can get it working!
